src/pages/settings/AppDownloadLinks.js

```diff
@@ -43,7 +42,7 @@ const AppDownloadLinksPage = ({translate}) => {
             <HeaderWithCloseButton
                 title={translate('initialSettingsPage.aboutPage.appDownloadLinks')}
                 shouldShowBackButton
-                onBackButtonPress={() => Navigation.goBack()}
+                onBackButtonPress={Navigation.goBack}
```

Comment:
NAB, use caution here, haven't tested this, but see `Navigation.goBack()` takes an optional boolean argument and events are truthy if passed.

Reply:
Probably we should update this method to do similar logic to this

https://github.com/Expensify/App/blob/bc3cfc2bee6a5e9e4ad9db46b8aac385a6164a85/src/libs/Navigation/Navigation.js#L136-L138

Reply:
Ah, good catch. I'd rather not pass the event so we are explicit about the arguments passed.